doc/examples/isometric/.gitignore

```diff
@@ -0,0 +1,70 @@
+# Miscellaneous
```

Comment:
@renancaraujo do we need this file?

Reply:
We already have one in the root for the examples right?